code/datums/Emote_system/emote.dm

```diff
@@ -428,13 +428,18 @@ VampyrBytes
 
 //Emote Cooldown System (it's so simple!)
 /datum/emote/proc/handle_emote_CD(var/mob/user)
-	if(user.emote_cd == 2) return 1			// Cooldown emotes were disabled by an admin, prevent use
-	if(user.emote_cd == 1) return 1		// Already on CD, prevent use
+	if(!user)
```

Comment:
Um, that's not possible. There is only one place `handle_emote_CD()` is called from, which is in `doEmote()`, and right before the call there is
```
if(!istype(user))
      return
```
So there should be no way for (!user) to be true